### PR TITLE
Update package.json/changelog for 3.3.1 release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,30 +40,29 @@
...
-    "core-js": "2.6.3",
-    "rxjs": "6.5.2",
-    "rxjs-compat": "6.5.2",
-    "ts-node": "8.3.0",
-    "tslint": "5.18.0",
+    "core-js": "2.6.9",
+    "rxjs": "6.5.3",
+    "rxjs-compat": "6.5.3",
+    "ts-node": "8.4.1",
+    "tslint": "5.20.0",
```
Comment (Blackbaud-AlexKingman) [inline package.json:65]: Why so many package upgrades?

Comment (Blackbaud-SteveBrush) [reply package.json:65]: Just development dependencies to fix some vulnerability warnings...

Comment (Blackbaud-AlexKingman) [reply package.json:65]: 👍

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Review by blackbaud-johnly — COMMENTED** [review]
LGTM

**Pull request merged**